SU-SWS/stanford_basic
# D8CORE-646: Add padding to top of page-title.

# READY FOR REVIEW

# Summary
- Adds padding to the Page Title on stanford-page content types.

# Review By (Date)
- October 25th

# Urgency
- Medium

# Steps to Test for D8CORE-646

1. Check out this branch
2. If you don't have a drupal install working do a fresh build (blt drupal:install)
3. If you do have a drupal install working clear all caches
4. Visit the `/resources` page and review the padding applied to the page title. 
5. Compile the CSS by running from the stanford_basic directory
```
nvm use
npm install
npm run publish
```

# Steps to test For D8CORE-630

- Do D8CORE-646 steps first
- Navigate to the home page
- Set your browser width so that the menu goes to the mobile option
- Click on the hamburger menu to drop the nav
- Ensure the drop down is over the hero banner's welcome text

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- D8CORE-646

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
